components/server/src/auth/bearer-authenticator.ts

```diff
@@ -103,6 +106,23 @@ export class BearerAuth {
         req.user = user;
     }
 
+    async tryAuthFromHeaders(headers: Headers): Promise<SubjectId | undefined> {
+        const token = getBearerToken(headers.get("authorization"));
+        log.debug("tryAuthFromHeaders", { token: new TrustedValue(token), headers: new TrustedValue(headers) });
```

Comment:
does it log value of token? I don't think we should if so

Reply:
although it is debug, and for debug logging I don't think we scrub, somehow i don't hink trustedvalue should be used here

Reply:
This is an oversight, just there for debugging. Will remove.

Reply:
Done.